examples/api-basic/main.go

```diff
@@ -17,20 +17,35 @@ limitations under the License.
 package main
 
 import (
+	"flag"
```

Comment:
I'd prefer using a separate example for metrics and profiling, to make it more focused.